pandas/core/generic.py

```diff
@@ -6428,6 +6428,15 @@ def astype(
             Return a copy when ``copy=True`` (be very careful setting
             ``copy=False`` as changes to values then may propagate to other
             pandas objects).
+
+            .. note::
```

Comment:
Can you make this a shared docstring?

Reply:
Personally I find shared docstrings quite annoying (and untransparant, difficult to understand), so while it's nice to deduplicate and only have 1 place to update this note, I am also fine with simply copy-pasting this note multiple times once we have finalized its wording.

(certainly for methods that right now don't yet make use of shared docstrings)

Reply:
Yeah I don't like them either